bar.m

```diff
@@ -38,76 +38,30 @@
 
 #pragma mark - Support Functions and Classes
 
-// Voodoo magic to swizzle _crashOnException because it seems that touchbar is sensitive (esp to
-// system wakeups) and throws exceptions even when it really shouldn't... see discussion at
-// https://github.com/asmagill/hs._asm.undocumented.touchbar/issues/1 and
-// https://nshipster.com/method-swizzling/
-@interface NSApplication (crashInterceptor)
-+ (void)xxx_hammerspoon_SwizzleCrashOnException ;
-- (void)xxx_hammerspoon_crashOnException:(NSException *)exception ;
-@end
-
-@implementation NSApplication (crashInterceptor)
+@implementation NSFunctionRow: NSObject
```

Comment:
This needs to be a category, not a class decfinition -- `@implementation NSFunctionRow (interceptor)`

Reply:
Fixed!